tests/ad/conftest.py

```diff
@@ -0,0 +1,27 @@
+'''conftest'''
+import os
+
+import pytest
+from tenable.ad.session import TenableAD
+
+
+@pytest.fixture(scope='module')
+def vcr_config():
+    '''vcr config fixture'''
+    return {
+        'filter_headers': [
+            ('X-API-Key', 'accessKey=TAD_API_KEY'),
+        ],
+    }
+
+
+@pytest.fixture
+def api():
+    '''api key fixture'''
+    return TenableAD(
+        api_keys = os.getenv('TAD_API_KEY', 'ffffffffffffffffffffffffffffffffffff'),
+        url='https://pytenable.tenable.ad',
+        ssl_verify=False,
+        vendor='pytest',
+        product='pytenable-automated-testing'
+    )
```

Comment:
As responses are being used instead of vcr, this should be safe to remove.

Reply:
files are carry forwarded from base branch and already addressed in another PR https://github.com/tenable/pyTenable/pull/466. Assuming to handle conflicts easily, these files are not modified in this PR